File: leetcode/january-leetcoding-challenge-2021/merge_sorted_array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        for (int i = nums1.length - 1, j = 0; i >= m; i--, j++) {
            nums1[i] = nums2[j];
        }
        Arrays.sort(nums1);
    }
}

// https://leetcode.com/explore/challenge/card/january-leetcoding-challenge-2021/580/week-2-january-8th-january-14th/3600/

File: leetcode/arrays-101/merge_sorted_array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        for (int i = nums1.size() - 1, j = 0; i >= m; i--, j++) {
            nums1[i] = nums2[j];
        }
        sort(nums1.begin(), nums1.end());
    }
};

// https://leetcode.com/explore/learn/card/fun-with-arrays/525/inserting-items-into-an-array/3253/